src/polynomial.cpp

```diff
@@ -0,0 +1,310 @@
+#include "polynomial.h"
+#include "add.h"
+#include "mul.h"
+#include "pow.h"
+#include "constants.h"
+
+namespace SymEngine {
+
+    Polynomial::Polynomial(const RCP<const Symbol> &var, map_uint_mpz&& dict) :
```

Comment:
Yeah, I would shift it left. Everything is in a namespace, so it would be a
waste of 4 spaces and vertical space is we indent it.

Sent from my mobile phone.
On Jun 4, 2015 6:06 AM, "Isuru Fernando" <notifications@github.com> wrote:

> In src/polynomial.cpp
> <https://github.com/sympy/symengine/pull/454#discussion_r31714058>:
>
> > @@ -0,0 +1,310 @@
> > +#include "polynomial.h"
> > +#include "add.h"
> > +#include "mul.h"
> > +#include "pow.h"
> > +#include "constants.h"
> > +
> > +namespace SymEngine {
> > +
> > +    Polynomial::Polynomial(const RCP<const Symbol> &var, map_uint_mpz&& dict) :
>
> Rest of SymEngine code does not use four spaces like this before
> Polynomial. @certik <https://github.com/certik>, should this be changed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/454/files#r31714058>.
>